### PR TITLE
[GEOS-9409] Change the wording ‘Boring default style’ in the Styles

### PR DESCRIPTION
**Pull request by imranrajjad**
-replaced word 'boring' with 'simple'
-improved description on styles inside code and sample data directories

<Include a few sentences describing the overall goals for this Pull Request>

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

For all pull requests:

- [x] Confirm you have read the [contribution guidelines](https://github.com/geoserver/geoserver/blob/master/CONTRIBUTING.md) 
- [x] You have sent a Contribution Licence Agreement (CLA) as necessary (not required for small changes, e.g., fixing typos in documentation)
- [x] Make sure the first PR targets the master branch, eventual backports will be managed later. This can be ignored if the PR is fixing an issue that only happens in a specific branch, but not in newer ones.

The following are required only for core and extension modules (they are welcomed, but not required, for community modules):
- [x] There is a ticket in Jira describing the issue/improvement/feature (a notable exemptions is, changes not visible to end users)
- [x] PR for bug fixes and small new features are presented as a single commit
- [x] Commit message must be in the form "[GEOS-XYZW] Title of the Jira ticket" (export to XML in Jira generates the message in this exact form)
- [ ] New unit tests have been added covering the changes
- [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [ ] This PR passes the [QA checks](https://docs.geoserver.org/latest/en/developer/qa-guide/index.html) (QA checks results will be reported by travis-ci after opening this PR)
- [ ] Commits changing the UI, existing user workflows, or adding new functionality, need to include documentation updates

Submitting the PR does not require you to check all items, but by the time it gets merged, they should be either satisfied or inapplicable.

### CODE REVIEW

File: data/app-schema-tutorial/styles/raster.sld [hunk 1]
```diff
@@ -5,7 +5,7 @@ http://schemas.opengis.net/sld/1.0.0/StyledLayerDescriptor.xsd" version="1.0.0">
 	<Name>raster_layer</Name>
 	<UserStyle>
 		<Name>raster</Name>
-		<Title>A boring default style</Title>
+		<Title>A sample default style</Title>
```
Comment (aaime) [inline data/app-schema-tutorial/styles/raster.sld]: Better, but not descriptive

File: data/citensg-1.0/styles/default_line.sld [hunk 2]
```diff
@@ -12,7 +12,7 @@
 		<UserStyle>
 		    <!-- they have names, titles and abstracts -->
 		  
-			<Title>A boring default style</Title>
+			<Title>Green Line</Title>
```
Comment (aaime) [inline data/citensg-1.0/styles/default_line.sld]: It's actually at blue line?

File: data/citensg-1.0/styles/default_point.sld [hunk 3]
```diff
@@ -12,7 +12,7 @@
 		<UserStyle>
 		    <!-- they have names, titles and abstracts -->
 		  
-			<Title>A boring default style</Title>
+			<Title>A simple default style</Title>
```
Comment (aaime) [inline data/citensg-1.0/styles/default_point.sld]: Title goes in legend, so please short descriptive text. "Red point" for example. (the abstract needs fixing too, the point is not purple). Can you fix the others accordingly too?
To check if the description matches the appearance use the map preview in the style editor of GeoServer, you'll get the legend overlayed with the map.

File: data/app-schema-tutorial/styles/raster.sld (hunk 1 above)
Comment (imranrajjad) [reply data/app-schema-tutorial/styles/raster.sld]: will improve

File: data/citensg-1.0/styles/default_line.sld (hunk 2 above)
Comment (imranrajjad) [reply data/citensg-1.0/styles/default_line.sld]: will fix

File: data/citensg-1.0/styles/default_point.sld (hunk 3 above)
Comment (imranrajjad) [reply data/citensg-1.0/styles/default_point.sld]: understood, will re check all.

**Comment by imranrajjad** [comment]
@aaime made the requested changes, please let me know if anything is wrong

File: data/app-schema-tutorial/styles/default_line.sld [hunk 4]
```diff
@@ -12,7 +12,7 @@
 		<UserStyle>
 		    <!-- they have names, titles and abstracts -->
 		  
-			<Title>A boring default style</Title>
+			<Title>Green Line</Title>
```
Comment (jodygarnett) [inline data/app-schema-tutorial/styles/default_line.sld]: Consistent title case between "Green Line" vs "Red Square point" etc... do you care?

Comment (aaime) [reply data/app-schema-tutorial/styles/default_line.sld]: Is it just me or the default line is blue, not green? :-D

Comment (imranrajjad) [reply data/app-schema-tutorial/styles/default_line.sld]: I will fix that

**Comment by imranrajjad** [comment]
@jodygarnett thank you  for the review

> A couple questions:
> 
> Do you care about consistency in description / title?
Yes
> Do you want to use the generated style rather than this?
Yes
> Is there any documentation to update (could do a quick search).
I did a general content search for the word : Boring, boring and did not find anything

File: data/citewcs-1.1/styles/raster.sld [hunk 5]
```diff
@@ -5,7 +5,7 @@ http://schemas.opengis.net/sld/1.0.0/StyledLayerDescriptor.xsd" version="1.0.0">
 	<Name>raster_layer</Name>
 	<UserStyle>
 		<Name>raster</Name>
-		<Title>A simple default style</Title>
+		<TitleOpaque Raster</Title>
```
Comment (imranrajjad) [inline data/citewcs-1.1/styles/raster.sld]: `<TitleOpaque Raster</Title>`

invalid XML

File: data/citewms-1.3/styles/Streams.sld [hunk 6]
```diff
@@ -4,14 +4,14 @@
...
-      <sld:Title>Default Styler</sld:Title>
+      <sld:Title>Blue Polygon</sld:Title>
       <sld:FeatureTypeStyle>
         <sld:Name>name</sld:Name>
         <sld:FeatureTypeName>Streams</sld:FeatureTypeName>
         <sld:Rule>
           <sld:Name>name</sld:Name>
-          <sld:Title>title</sld:Title>
-          <sld:Abstract>Abstract</sld:Abstract>
+          <sld:Title>>Blue Polygon</sld:Title>
```
Comment (imranrajjad) [inline data/citewms-1.3/styles/Streams.sld]: invalid xml

**Comment by imranrajjad** [comment]
@aaime rebased and now build errors have gone away

**Comment by jodygarnett** [comment]
Thanks for checking the docs, there are probably screen snap examples but those can be updated over time.

**Comment by imranrajjad** [comment]
backport to 2.16.x https://github.com/geoserver/geoserver/pull/3919

**Pull request merged**